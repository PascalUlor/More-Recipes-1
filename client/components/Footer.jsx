import React from 'react';

const Footer = () => (
  <footer>
    <div className="container">
      <div className="row mt-4">
        <div className="col-12 col-sm-8 col-md-9 col-lg-10">
          <small id="copywrite" className="text-white"><span className="text-warning">&copy;2018 </span>
          Bootcamp27, Andela Nigeria. All rights reserved.</small>
        </div>
        <div className="col-12 col-sm-4 col-md-3 col-lg-2 text-center">
          <ul className="list-inline">
            <li className="list-inline-item">
              <a href="#">
                <i className="fa fa-google-plus" aria-hidden="true"></i>
              </a>
            </li>
            <li className="list-inline-item">
              <a href="#">
                <i className="fa fa-github"></i>
              </a>
            </li>
            <li className="list-inline-item">
              <a href="#">
                <i className="fa fa-facebook"></i>
              </a>
            </li>
            <li className="list-inline-item">
              <a href="#">
                <i className="fa fa-twitter"></i>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
);

export default Footer;