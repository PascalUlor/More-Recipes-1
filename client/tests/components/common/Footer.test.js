/* eslint-disable */
import React from 'react';
import toJson from 'enzyme-to-json';
import Footer from '../../../components/Footer.jsx';


describe('<Footer/>', () => {
  it('renders footer component without crashing', () => {
    const shallowWrapper = shallow(<Footer/>);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
