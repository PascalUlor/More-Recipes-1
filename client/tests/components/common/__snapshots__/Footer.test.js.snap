// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders footer component without crashing 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="row mt-4"
    >
      <div
        className="col-12 col-sm-8 col-md-9 col-lg-10"
      >
        <small
          className="text-white"
          id="copywrite"
        >
          <span
            className="text-warning"
          >
            Â©2018 
          </span>
          Bootcamp27, Andela Nigeria. All rights reserved.
        </small>
      </div>
      <div
        className="col-12 col-sm-4 col-md-3 col-lg-2 text-center"
      >
        <ul
          className="list-inline"
        >
          <li
            className="list-inline-item"
          >
            <a
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-google-plus"
              />
            </a>
          </li>
          <li
            className="list-inline-item"
          >
            <a
              href="#"
            >
              <i
                className="fa fa-github"
              />
            </a>
          </li>
          <li
            className="list-inline-item"
          >
            <a
              href="#"
            >
              <i
                className="fa fa-facebook"
              />
            </a>
          </li>
          <li
            className="list-inline-item"
          >
            <a
              href="#"
            >
              <i
                className="fa fa-twitter"
              />
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</footer>
`;
